Dragon.h

```diff
@@ -1,11 +1,20 @@
 #pragma once
-
 #include "Character.h"
+#include "Utility.h"
 
 struct Dragon : Character
 {
+    Dragon (std::string name_, int hp, int armor_, int attackDamage_ = 80 ) : 
+    Character( hp, armor_, attackDamage_ ),
+    name (name_)
+    {
+        
+    }
+    
+    const std::string& getName() override { return name; }
+    std::string getStats() override { return getCharacterStats(this); }
```

Comment:
These should all be implemented in the cpp file, not the header file.
- [ ] revise